wcomponents-core/src/main/java/com/github/bordertech/wcomponents/DefaultWComponent.java

```diff
@@ -9,8 +9,6 @@
 public final class DefaultWComponent extends AbstractWComponent {
 
 	/**
-	 * {@inheritDoc}
```

Comment:
Please retain this inheritDoc tag as the inheritance needs to be explicit due to the other comments in the block.